src/jennifer/query_builder/query.cr

```diff
@@ -301,6 +301,21 @@ module Jennifer
         self
       end
 
+      # Mutates query applying all modification returned from the block.
+      #
+      # Yields the expression builder and block is also executed with expression builder context.
+      #
+      # ```
+      # User.where(email: "example@test.com")
+      # ```
+      def where(**opts)
+        data = self
+        opts.each do |k, v|
+          data = data.where { c(k.to_s) == v }
```

Comment:
we can avoid N invocation of `#where` here iterating over key-values inside of the block; also `data` variable seems unnecessary here as it is just an alias for `self`